### ISSUE TITLE
Not working properly

### ISSUE THREAD

**Issue by rawnly** [jrmoulton/tmux-sessionizer#15]
Maybe it's my fault due to misconfiguration but when i run `tms` it's stuck without showing results.

[video demo](https://share.cleanshot.com/NfQB9H9b)

heres' my config: 

```toml
search_paths = ['/Users/federicovitale/Developer']
excluded_dirs = ['/Users/federicovitale/Developer/PYTHON']
default_session = 'wez-main'
```

**Comment by jrmoulton** [comment]
I'm guessing that it is working but that there are a lot of files and folders. Tmux sessionizer doesn't support a recursion limit yet (should be easy to add but I haven't done it yet) but I think that would fix the problem. Do you know if you have any really large folders in there? Like node modules maybe? If so patterns like node_modules can be excluded

**Comment by rawnly** [comment]
Yep definitely that. Maybe following the .gitignore specs could help, caching the list could be a great solution too.
Even for pure rust projects it takes ~10s to load with `--excluded target`

**Comment by Nitro01010** [comment]
I have a folder for all my programming projects named "Dev", and folders for languages inside there. So a python project would be inside "~/Dev/python/python-test". If you set the config like this: `tms config -p ~/Dev/*/*/` and then run `tms`, it's much faster than just doing it with the full folder, and it's really easy now that I've setup an alias.

**Comment by btctrl** [comment]
Guys where do I write a config instead of configuring paths through the CLI?

Will it read it in $XDG_CONFIG_HOME?
I cannot get `tms` to read multiple paths with `--paths` 

Let me know if I should create another issue to better track

**Comment by jrmoulton** [comment]
should be fixed in #20
